doc/nrf/external_comp/nrf_cloud.rst

```diff
@@ -129,6 +147,10 @@ The following application uses the :ref:`lib_nrf_cloud` for services in |NCS|:
 
 * :ref:`asset_tracker_v2`
 
+The following sample demonstrates nRF Cloud-specific functionality using CoAP:
+
+* To be determined
```

Comment:
Yes, but that sample is not yet (in the git history) available, so the docs build fails.  I have removed this for now, but have add the modem_shell as a sample to this section, since Tommi Kangas updated that and a location library update for CoAP to a PR build on top of this PR.  I will then add my CoAP sample to this same section in my sample's specific PR, built on top of both of those PRs.